exercises/handling-errors/README.md

```diff
@@ -0,0 +1,63 @@
+## Exercise #1: Handling Errors
+
+During this exercise, you will:
+
+- Throw and handle exceptions in Temporal Workflows and Activities
+- Use non-retryable errors to fail an Activity
+- Locate the details of a failure in Temporal Workflows and Activities in the Event History
+
+Make your changes to the code in the `practice` subdirectory (look for `TODO` comments that will guide you to where you should make changes to the code). If you need a hint or want to verify your changes, look at the complete version in the `solution` subdirectory.
+
+## Setup
+
+You'll need two terminal windows for this exercise.
+
+1. In all terminals, change to the `exercises/handling-errors/practice` directory.
+2. In one terminal, run `npm install` to install dependencies.
+
+## Part A: Defining Errors for Activities
+
+In this part of the exercise, you will throw an Application Failure that will fail your Activities.
+
+Application Failures are used to communicate application-specific failures in Workflows and Activities. In Workflows, if you throw an `ApplicationFailure`, the Workflow Execution will fail. In Activities, you can either throw an `ApplicationFailure` or another Error to fail the Activity Task. 
+
+1. Edit the `activities.ts` file.
+2. Import `ApplicationFailure` from `@temporalio/common`.
+3. In the `sendBill` Activity, notice how we throw an Application Failure error if the charge is negative. The Application Failure includes a message as well as a details array with a list of details pertaining to the failure. Since we want our custom error data to be serialized and transmitted over the network, we must set our custom data in the `details` field. 
+4. Go to the `validateCreditCard` Activity. We want to throw an error if the entered credit card number does not have 16 digits. Replace the logged error with your own `ApplicationFailure`, following the pattern you saw in step 3.
+
+## Part B: Throw the Activity Failures in Your Workflow
+
+In this part of the exercise, you will throw your Activity Failures that will fail your Workflow Executions.
```

Comment:
This is a part that I'm unsure about. If an Activity Failure is a specific failure, I'd go ahead and use code face and the actual name of it (for example `ActivityFailure`). I don't know if Activity Failure is one of the "blessed" words. It doesn't appear in the Vale list (which I know is incomplete). If you want to use it in a non-specific way, maybe use "error"?

Also, I have some suggestions about tightening prose. I was going to present it at a group meeting a few weeks ago, but things got a bit crazy. I can demo for you directly or in a sync or whatever works.